extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/MultiPartConfig.java

```diff
@@ -19,6 +19,5 @@ public interface MultiPartConfig {
      *
      * For now, this setting only works when using RESTEasy Reactive.
      */
-    @WithConverter(TrimmedStringConverter.class)
-    Optional<List<String>> fileContentTypes();
+    Optional<List<@WithConverter(TrimmedStringConverter.class) String>> fileContentTypes();
```

Comment:
Just to be sure, did you check we don't have other issues in this patch?

Reply:
Yes, I did and didn't spot any more related issues.